gamestonk_terminal/stocks/fundamental_analysis/dcf_view.py

```diff
@@ -51,19 +51,19 @@ def dcf(other_args: List[str], ticker: str):
         help="Confirms that the numbers provided are accurate.",
     )
 
-    try:
-        ns_parser = parse_known_args_and_warn(parser, other_args)
-        if not ns_parser:
-            return
-
-        dcf_view = CreateExcelFA(ticker, ns_parser.audit)
-        trypath = dcf_view.create_workbook()
-        print(
-            f"Analysis successfully ran for {ticker}\nPlease look in {trypath} for the file.\n"
-        )
+    # try:
```

Comment:
You should keep the try here, to avoid the terminal crashing in case there's an issue